govcd/vm.go

```diff
@@ -555,3 +555,19 @@ func (vm *VM) AnswerQuestion(questionId string, choiceId int) error {
 	return vm.client.ExecuteRequestWithoutResponse(apiEndpoint.String(), http.MethodPost,
 		"", "error asnwering question: %s", answer)
 }
+
+// ToggleHardwareVirtualization allows to either enable or disable hardware assisted
+// CPU virtualization for the VM. It can only be performed on a powered off VM and
+// will return an error otherwise. This is mainly useful for hypervisor nesting.
+func (vm *VM) ToggleHardwareVirtualization(isEnabled bool) (Task, error) {
+	apiEndpoint, _ := url.ParseRequestURI(vm.VM.HREF)
```

Comment:
I think it's safer to add check that VM is turned off and if not return explicit message